### DISCUSSION TITLE
Esp8266 DIY possible?

### DISCUSSION THREAD

**Discussion by mlaass** [meshtastic/firmware#3712] [Q&A] [answered]
I have several devices based on esp8266 with LoRa shields and I thought it would be cool if I could run Meshtastic on them as they are cheap, low power, and similar to the Esp32.
A lot of features should be replicable, such as wifi networking, serial configuration, and even Bluetooth might be an option.
In a naïve attempt, I tried to get the firmware to compile for esp8266 but so far without success.
It seems there are a lot of places where the preprocessor is used for conditional compilation based on the assumed architecture.
I am new to the codebase, so I eventually got overwhelmed by the task.
I was wondering if there is any interest in a port to esp8266 and if there are others who maybe have some experience that could help me break down the task into manageable steps.
I'd appreciate any response.

**Comment by garthvh** [comment, marked as answer]
ESP8266 is not supported and has been discontinued
